blocks/form/file.js

```diff
@@ -0,0 +1,52 @@
+function attachFiles(wrapper, fileList, fileTemplate) {
+    for(let i = 0; i < fileList.length; i++) {
+        const file = fileList[i];
+        const input = fileTemplate.cloneNode(true);
+        input.id = input.name = 'attachment_' + Date.now() + '' + Math.floor((Math.random() * 1000));
```

Comment:
you could reduce the risk of collisions even more by  converting to a base-36 string:
```
function generateUniqueId() {
    return (Date.now().toString(36) + Math.random().toString(36).substr(2, 5)).toUpperCase();
}
```

and then use as:
```
input.id = input.name = 'attachment_' + generateUniqueId();
```

Reply:
Removed the usage of generating unique, using only one input.